Tests/Form/FormFactoryTest.php

```diff
@@ -37,30 +48,24 @@ public function testIsCreatingASimpleForm()
             ],
         ];
 
-        $formData = 'form_foo_data';
-        $formOptions = 'form_foo_options';
+        $this->formFactoryMock->createNamedBuilder('foo', 'form', ['foo_form_data'], ['foo_form_options'])
+            ->shouldBeCalled()
```

Comment:
You should not be using both `shouldBeCalled()` and `willReturn()`. If you use `willReturn()`, `shouldBeCalled()` is not needed.